library/src/main/java/com/schibsted/spain/barista/rule/BaristaRule.kt

```diff
@@ -0,0 +1,63 @@
+package com.schibsted.spain.barista.rule
+
+import android.app.Activity
+import android.content.Intent
+import android.support.test.rule.ActivityTestRule
+import com.schibsted.spain.barista.rule.cleardata.ClearDatabaseRule
+import com.schibsted.spain.barista.rule.cleardata.ClearFilesRule
+import com.schibsted.spain.barista.rule.cleardata.ClearPreferencesRule
+import com.schibsted.spain.barista.rule.flaky.FlakyTestRule
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class BaristaRule<T : Activity> private constructor(activityClass: Class<T>) : TestRule {
+
+    private val clearPreferencesRule: ClearPreferencesRule = ClearPreferencesRule()
+    private val clearDatabaseRule: ClearDatabaseRule = ClearDatabaseRule()
+    private val clearFilesRule: ClearFilesRule = ClearFilesRule()
+    private val flakyTestRule: FlakyTestRule
+    private val activityTestRule: ActivityTestRule<T>
+
+    init {
+        this.flakyTestRule = FlakyTestRule()
```

Comment:
Why don't just join these initialisations with the declaration? Just like the other ones